src/pages/spaces/workspace/hooks/useWorkspaceKpiCards.tsx

```
@@ -208,18 +228,21 @@ export function useWorkspaceKpiCards(params: {
-  headerKpiCardsRef.current = headerKpiCards;
+  useEffect(() => {
+    const bump = () => setTasksRefreshKey((prev) => prev + 1);
+    const unsubscribers = [
+      TaskEvents.on(TaskEvents.EVENTS.TASK_CREATED, bump),
+      TaskEvents.on(TaskEvents.EVENTS.TASK_UPDATED, bump),
+      TaskEvents.on(TaskEvents.EVENTS.TASK_DELETED, bump),
+      TaskEvents.on(TaskEvents.EVENTS.TASKS_BULK_UPDATE, bump),
+      TaskEvents.on(TaskEvents.EVENTS.CACHE_INVALIDATE, bump),
+    ];
+    return () => {
+      unsubscribers.forEach((u) => u());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)